.circleci/config.yml

```diff
@@ -329,6 +339,107 @@ jobs:
           success_message: ":rocket: RC version ${STREAMLIT_RELEASE_VERSION} was successful!"
           failure_message: ":blobonfire: RC version ${STREAMLIT_RELEASE_VERSION} failed to release"
 
+  build-deploy-release:
+    resource_class: large
+    docker:
+      - image: circleci/python:3.9.7
+    working_directory: ~/repo
+    steps:
+      - checkout:
+          name: Checkout Streamlit code
+
+      - run:
+          name: Install GitHub CLI tool and related dependencies
+          command: |
+            sudo apt-get install jq
+            /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+            eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+            brew install gh
+
+      - run:
+          name: Configure GitHub CLI Settings
+          command: echo 'export GH_REPO=$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME' >> $BASH_ENV
```

Comment:
Should we use hard-code the streamlit repo? I imagine that we want releases to be created in the main repo and not forks.

Reply:
My gut feeling is that it doesn't matter, because who is going to do releases in a fork? If anything, it seems better to leave it as is, to avoid interference between forks and the main repo.